content/docs/hooks-reference.md

```diff
@@ -448,7 +448,7 @@ function FancyInput(props, ref) {
       inputRef.current.focus();
     }
   }));
-  return <input ref={inputRef} ... />;
+  return <input ref={inputRef} {...props} />;
```

Comment:
Whether or not you should spread props has different opinions. Since this is not relevant to `useImperativeHandle` we could just omit it and avoid opinions?

Reply:
Agree. I thought someone will confuse `...` represent means in the here.

Feel free to close PR, if don't need it, thank you for reviewing.